src/main/java/app/ticket/ticketing/concert/ConcertRepository.java

```diff
@@ -0,0 +1,20 @@
+package app.ticket.ticketing.concert;
+
+import app.ticket.ticketing.db.Concert;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface ConcertRepository extends JpaRepository<Concert, Long> {
+    Concert findByConcertId(String concertId);
+
+    @Query("SELECT concertId FROM Concert c WHERE c.concertId = :concertId AND c.isDelete = FALSE")
+    Optional<Concert> findByConcertIdIdForDelete(@Param("concertId") String concertId);
+
+    @Query("UPDATE Concert c SET c.isDelete = :isDelete WHERE c.concertId = :concertId")
+    void deleteByConcertId(@Param("concertId") String concertId, @Param("isDelete") Boolean isDelete);
```

Comment:
```suggestion
    @Query("UPDATE Concert c SET c.isDelete = TRUE WHERE c.concertId = :concertId")
    void deleteByConcertId(@Param("concertId") String concertId;
```
삭제 메소드라면 isDelete 를 받지 않고 True 로 해도 되지 않을까요?

Reply:
삭제 메소드에서 isDelete는 불필요할 것 같아 수정하도록 할께요.